### ISSUE TITLE
Airflow 2.2.0 LDAP integration broken with TLS disabled

### ISSUE THREAD

**Issue by eclipsetalk** [bitnami/bitnami-docker-airflow#112]
Hello,
When I disable TSL for LDAP with Airflow 2.2.0. I get the following error.
```
File "/opt/bitnami/airflow/webserver_config.py", line 47, in <module>
    AUTH_TYPE = AUTH_LDAP
NameError: name 'AUTH_LDAP' is not defined
```

The package python-ldap is installed.

Here is my ldap definition in the chart:
```
ldap:
  enabled: true
  uri: "ldaps://xxx.corp:636"
  base: "DC=xx,DC=CORP"
  uidField: 'cn'
  binddn: "CN=user,OU=Svcaccts,DC=xx,DC=CORP"
  userRegistration: 'True'
  userRegistrationRole: 'Public'
  rolesMapping: '{ "cn=All,ou=Groups,dc=example,dc=org": ["User"], " CN=ml_ops,OU=Security,OU=Groups,OU=DVA-US,DC=xx,DC=Corp": ["Admin"], }'
  rolesSyncAtLogin: 'True'

  tls:
    enabled: false
    allowSelfSigned: "'True'"
    CAcertificateSecret: ''
    CAcertificateFilename: ''
```

**Comment by marcosbc** [comment]
Hi @eclipsetalk, you should get a notification from this thread once the original issue is closed. If not, you can always 'watch' the other issue in the GitHub UI.

**Comment by ITJamie** [comment]
@marcosbc any idea when that PR is going to be merged?

**Comment by marcosbc** [comment]
@ITJamie Unfortunately not. However I recommend to continue the discussion in the PR. If it is blocked by the reviewer, just sending a ping or ask to have it reviewed again should be enough.

**Comment by eclipsetalk** [comment]
Any updates on this PR? Thx

**Comment by marcosbc** [comment]
@eclipsetalk The PR was just merged, the tag `2.2.2-debian-10-r7` should have the fix.
